transformato/charmm_factory.py

```diff
@@ -31,8 +31,16 @@ def charmm_factory(configuration: dict, structure: str, env: str):
     nstout = configuration["simulation"]["parameters"]["nstout"]
     nstdcd = configuration["simulation"]["parameters"]["nstdcd"]
     steps_for_equilibration = configuration["solvation"]["steps_for_equilibration"]
-    GPU = True
-    switch = "vfswitch"
+    try:
+        GPU = configuration["simulation"]["parameters"]["GPU"]
+    except:
```

Comment:
if possible always except the most specific error that is to be expected. In this case it's a KeyError()